docs/installation.md

```diff
@@ -0,0 +1,243 @@
+# Installing ansible-navigator with execution environment support
+
+[podman_url]: https://podman.io/
+
+```{contents}
+:local:
+```
+
+## Linux
+
+[podman_install_url]: https://podman.io/getting-started/installation
+[docker_for_linux_url]: https://docs.docker.com/engine/install/
+
+### Requirements
+
+- Either [podman][podman_url] or [Docker for Linux][docker_for_linux_url]
+- Internet access (during initial installation)
+
+### Install the desired container engine for execution environment support
+
+- Follow the [podman installation instructions][podman_install_url] for the appropriate distribution.
+- Follow the [Docker for Linux installation instructions][docker_for_linux_url] for the appropriate distribution.
+
+### Install ansible-navigator
+
+1. Install the python package manager using the system package installer (e.g.):
+
+   ```
```

Comment:
I'd prefer the code blocks to have explicit highlighting. We could go for `bash` or alternatively, we could go for `console`. But the latter is used for CLI recordings with a prompt like `$` prepended to the commands followed by their example outputs.

Reply:
Also, it seems like the copybutton extension is able to cut the prompts off.